### PR TITLE
update install instructions

### PR DESCRIPTION
**Pull request by ocefpaf**
Closes https://github.com/ioos/ioos_code_lab/issues/54

### CODE REVIEW

**Comment by ocefpaf** [comment]
@MathewBiddle do you mind taking a look at the changes here? If you could try to follow them and check if they are clear enough, even better :-)

**Comment by MathewBiddle** [comment]
Added some comments in specific lines. 

With all the hyperlinks I was wondering if there was a GHA to check links on a .md page (or from the .html in `gh-pages` branch). It would be nice to get a notification if/when links break in our documentation.

**Comment by ocefpaf** [comment]
> With all the hyperlinks I was wondering if there was a GHA to check links on a .md page (or from the .html in `gh-pages` branch). It would be nice to get a notification if/when links break in our documentation.

Sphinx has a check link tool. I'll see if that is available on Jupyter book too.

**Comment by ocefpaf** [comment]
> Added some comments in specific lines.

If you made it like a "review" did you submit it? I cannot see the comments yet.

**Comment by MathewBiddle** [comment]
Sorry about that, just submitted it.

File: jupyterbook/content/ioos_installation_conda.md [hunk 1]
```diff
@@ -142,3 +159,11 @@ conda-lock -f environment.yml -p osx-64 -p linux-64 -p win-64
 ```
 
 In the example above we are locking for macOS, Linux and Windows.
+
+### Trouble-shooting Windows
+
+Windows can be complicated and here are a few tips to help you troubleshoot it:
```
[the remark this answers is not in the document]
Comment (ocefpaf) [reply jupyterbook/content/ioos_installation_conda.md:165]: That's the first item but I guess it is not clear them :-/

File: jupyterbook/content/ioos_installation_conda.md [hunk 2]
```diff
@@ -1,53 +1,51 @@
 # Installing the IOOS conda environment
 
-For IOOS python users we recommend the free
-[Miniconda](http://conda.pydata.org/miniconda.html) Python distribution,
-a lightweight version of the [Anaconda Scientific Python Distribution](https://store.continuum.io/cshop/anaconda/).
+For IOOS Python/R users we recommend the free
+[Miniforge](https://github.com/conda-forge/miniforge) distribution,
+a lightweight version of the [Anaconda Scientific Python Distribution](https://store.continuum.io/cshop/anaconda/) with the conda-forge channel pre-configured.
```
[the remark this answers is not in the document]
Comment (ocefpaf) [reply jupyterbook/content/ioos_installation_conda.md:5]: > So, now we recommend Miniforge?

Yep. Mostly b/c of: https://www.anaconda.com/terms-of-service

> Do we still want to include Miniconda information in here as well?

Everything is virtually identical. Not sure it is worth mentioning it. If folks need miniconda for any reason they can use this instructions. The only differences are that they need to add the conda-forge channel manually and activate strict channel option.

> Does it make a difference to the end user?

Besides the new TOS the difference are that they need to add the conda-forge and activate strict channel.

File: jupyterbook/content/ioos_installation_conda.md [hunk 3]
```diff
@@ -123,11 +123,28 @@ Check out [Travis Oliphant's blog piece](http://technicaldiscovery.blogspot.com/
 - Raise an issue [here](https://github.com/ioos/ioos_code_lab/issues)
 - Please get help on the [IOOS-tech Google Group](https://groups.google.com/forum/?hl=en#!forum/ioos_tech)
 
-## Appendix: conda-lock
+## Appendix:
+
+### What to do when everything seems to be broken?
+
+1. If you believe that only your environment is broken you can follow the [update environment](##Updating-the-IOOS-environment) instructions from above;
```
[the remark this answers is not in the document]
Comment (ocefpaf) [reply jupyterbook/content/ioos_installation_conda.md]: Thanks! Vscode fooled me there by creating the link that didn't point to the right section :-)

File: jupyterbook/content/ioos_installation_conda.md [hunk 4]
```diff
@@ -123,11 +123,28 @@ Check out [Travis Oliphant's blog piece](http://technicaldiscovery.blogspot.com/
...
+1. Sometimes conda updates can break backwards compatibility and updating is broken. In those cases remove the Miniforge3 directory and perform a fresh install of the new version.
+1. In rare cases you may want to install a frozen version of the environment. Like, you need the exact same version that is running on our CIs. You can accomplish that by [downloading the lock file your your system](https://github.com/ioos/ioos_code_lab/tree/main/.binder) and issuing the command:
```
[the remark this answers is not in the document]
Comment (ocefpaf) [reply jupyterbook/content/ioos_installation_conda.md]: > Might want to include a file name description/explanation for the lock file, so folks know which to download.

I tried to make it a bit clearly below. Let me know if that is better.

**Review by MathewBiddle — APPROVED** [review]
Looks good.

**Pull request merged**